### PR TITLE
fix makeTransparent

### PR DESCRIPTION
**Pull request by strausr**
#### What does this PR solve?
fix makeTransparent fromJson to return e_make_transparent rather than e_makeTransparent

### CODE REVIEW

File: src/actions/effect/leveled/MakeTransparent.ts [hunk 1]
```diff
@@ -40,7 +41,7 @@ class MakeTransparentEffectAction extends LeveledEffectAction {
 
     // We are using this() to allow inheriting classes to use super.fromJson.apply(this, [actionModel])
     // This allows the inheriting classes to determine the class to be created
-    const result = new this(actionType, tolerance);
+    const result = new this(ACTION_TYPE_TO_EFFECT_MODE_MAP[actionType], tolerance);
```
Comment (const-cloudinary) [inline src/actions/effect/leveled/MakeTransparent.ts:44]: @strausr , how come this change does not affect other tests? like `unsharpMask`, `advancedRedEye`, etc?

Comment (const-cloudinary) [reply src/actions/effect/leveled/MakeTransparent.ts:44]: if it does, maybe it's reasonable to add other tests as well?

**Review by const-cloudinary — APPROVED** [review]

File: src/actions/effect/leveled/MakeTransparent.ts (hunk 1 above)
Comment (patrick-tolosa) [reply src/actions/effect/leveled/MakeTransparent.ts:44]: The reason is that those variables were already handled in our internal mapping 
(See src/internal/internalConstants.ts)

This PR added `makeTransparent: 'make_transparent'`

Comment (const-cloudinary) [reply src/actions/effect/leveled/MakeTransparent.ts:44]: OK

**Pull request merged**